src/main/java/trello/api/rest/RestClientAPI.java

```diff
@@ -12,57 +12,106 @@
 
 package trello.api.rest;
 
-import core.selenium.util.JsonConverter;
-import core.selenium.util.ReadJsonFile;
-import io.restassured.RestAssured;
-import io.restassured.builder.RequestSpecBuilder;
+import com.google.gson.Gson;
+import io.restassured.response.Response;
 import io.restassured.specification.RequestSpecification;
-import trello.entities.User;
-import trello.keys.NamePages;
+
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.http.ContentType.JSON;
 
 /**
- * RestClientAPI class.
+ * Authentication class.
```

Comment:
Please, change the name of this class we are in the RestClientAPI.

Reply:
Solved